package.json

```diff
@@ -24,19 +24,20 @@
   "scripts": {
     "build": "vite build",
     "test:code": "jest --collect-coverage",
-    "test:types": "jest -c jest.config.tsd.js --collect-coverage",
+    "test:types": "jest -c jest.config.tsd.js",
     "test": "npm run test:types && npm run test:code",
     "examples:app": "cd examples/app && npx vite build && node ./dist/di.es.js",
     "examples:hello": "cd examples/hello && npx vite build && node ./dist/di.es.js",
     "prepublishOnly": "npm run test && npm run build"
   },
   "devDependencies": {
+    "@tsd/typescript": "^5.1.3",
```

Comment:
You can install any version `@tsd/typescript`. This is the actual TypeScript package which will be used for type testing. So you can run the type tests on any target version.